test/run_test.py

```diff
@@ -25,6 +26,125 @@
 from typing import Dict, Optional, List
 
 REPO_ROOT = pathlib.Path(__file__).resolve().parent.parent
+TORCH_INSTALL_DIR = site.getsitepackages()[0] + "/torch"
+TORCH_BIN_DIR = TORCH_INSTALL_DIR + "/bin"
+TORCH_LIB_DIR = TORCH_INSTALL_DIR + "/lib"
+TORCH_TEST_DIR = TORCH_INSTALL_DIR + "/test"
```

Comment:
Please avoid string concat for path manupulation, but instead either use a `Path` class or `os.path.join`
```suggestion
TORCH_INSTALL_DIR = os.path.join(site.getsitepackages()[0], "torch")
TORCH_BIN_DIR = os.path.join(TORCH_INSTALL_DIR, "bin")
TORCH_LIB_DIR = os.path.join(TORCH_INSTALL_DIR, "lib")
TORCH_TEST_DIR = os.path.join(TORCH_INSTALL_DIR, "test")
```